Assignment3.java

```diff
@@ -0,0 +1,70 @@
+import java.io.IOException;
```

Comment:
You can add the bin folder into .gitignore so these class file changes will not show.